src/diagnostic-channel-publishers/tests/winston.spec.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+import * as assert from "assert";
+import {channel, IStandardEvent, makePatchingRequire} from "diagnostic-channel";
+
+import {enable as enableWinston, IWinstonData} from "../src/winston.pub";
+
+function compareWinstonData(actual: IWinstonData, expected: IWinstonData): void {
+    assert(actual.message === expected.message, "messages are not equal");
+    // meta is an object, but we can always use the same reference
+    assert(actual.meta === expected.meta, "meta objects are not equal");
+    assert(actual.level === expected.level, "levels are not equal");
+}
+
+describe("winston", () => {
+    let winston;
+
+    before(() => {
+        enableWinston();
+        winston = require("winston");
+    });
+
+    afterEach(() => {
+        (channel as any).reset();
+    });
+
+    it("should intercept the default logger", () => {
+        let actual: IWinstonData = null;
+        const expected: IWinstonData = {message: "should intercept the default logger", meta: {}, level: "info"};
+
+        channel.subscribe<IWinstonData>("winston", (event) => {
```

Comment:
To avoid repetition, you could hoist `actual` to the `describe` scope, and configure the subscriber in `beforeEach`